### DISCUSSION TITLE
Unable to render code block

### DISCUSSION THREAD

**Discussion by kishore0420** [community/community#75544] [New to GitHub]
when I am trying to access any of the pdf files, it is showing that "Unable to render code block" and the download option is also not working. The same problem is happening if I am trying to access repositories of other users where pdf files are uploaded.

**Comment by AidenJamesYt** [comment]
try the following steps:
1. Open the settings for your browser.
2. Locate the Privacy and Security section.
3. Click on the Content Settings option.
4. Scroll down to the PDF Documents section.
5. Click on the "Allow all sites to display PDF documents" option.

if this doesnt work, lmk (let me know)

**Reply by anshu15183** [reply]
not working, please help further

**Reply by adesoji-jpg** [reply]
how to upload a files that has shape on github please help futher
